dbos-config.schema.json

```diff
@@ -4,6 +4,18 @@
   "type": "object",
   "additionalProperties": false,
   "properties": {
+    "name": {
+      "type": "string",
+      "description": "The name of your application"
+    },
+    "language": {
+      "type": "string",
+      "description": "The language used in your application",
```

Comment:
Can we set a default language? It's annoying to add a new field everywhere.

Reply:
TS is the default